docs/samples/custom/timeseries-holtwinter/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use the official lightweight Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7-slim
+
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+
+# Install production dependencies.
+COPY requirements.txt ./
+RUN pip3 install --no-cache-dir -r ./requirements.txt
+
+# Copy local code to container image
+COPY model.py  model_holt.sav   ./
```

Comment:
not sure if it is a good practice to store model in the image  but I guess it is fine for an example.